### ISSUE TITLE
Exception thrown in IntelliJ Ultimate 13.1

### ISSUE THREAD

**Issue by tommahieu** [cefolger/needsmoredojo#163]
I've upgraded to IntelliJ Ultimate 13.1.  Needs More Dojo throws the following exceptions since the upgrade.

``` java
com.intellij.lang.javascript.psi.JSFunctionExpression.getFunction()Lcom/intellij/lang/javascript/psi/JSFunction;: com.intellij.lang.javascript.psi.JSFunctionExpression.getFunction()Lcom/intellij/lang/javascript/psi/JSFunction;
java.lang.NoSuchMethodError: com.intellij.lang.javascript.psi.JSFunctionExpression.getFunction()Lcom/intellij/lang/javascript/psi/JSFunction;
    at com.chrisfolger.needsmoredojo.core.amd.define.DefineResolver.addDefinesAndParametersOfImportBlock(DefineResolver.java:50)
    at com.chrisfolger.needsmoredojo.core.amd.importing.UnusedImportsRemover.filterUsedModules(UnusedImportsRemover.java:135)
    at com.chrisfolger.needsmoredojo.intellij.inspections.UnusedImportsInspection.checkFile(UnusedImportsInspection.java:72)
    at com.intellij.codeInspection.LocalInspectionTool$1.visitFile(LocalInspectionTool.java:144)
    at com.intellij.extapi.psi.PsiFileBase.accept(PsiFileBase.java:70)
    at com.intellij.lang.javascript.psi.impl.JSFileImpl.accept(JSFileImpl.java:120)
    at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:74)
    at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass$5.process(LocalInspectionsPass.java:380)
    at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass$5.process(LocalInspectionsPass.java:375)
    at com.intellij.concurrency.ApplierCompleter.a(ApplierCompleter.java:119)
    at com.intellij.concurrency.ApplierCompleter.c(ApplierCompleter.java:197)
    at com.intellij.concurrency.ApplierCompleter.a(ApplierCompleter.java:135)
    at com.intellij.concurrency.ApplierCompleter.access$000(ApplierCompleter.java:42)
    at com.intellij.concurrency.ApplierCompleter$1.run(ApplierCompleter.java:82)
    at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1153)
    at com.intellij.concurrency.ApplierCompleter$2.run(ApplierCompleter.java:91)
    at com.intellij.openapi.progress.ProgressManager.executeProcessUnderProgress(ProgressManager.java:209)
    at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:212)
    at com.intellij.concurrency.ApplierCompleter.a(ApplierCompleter.java:103)
    at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:79)
    at jsr166e.CountedCompleter.exec(CountedCompleter.java:684)
    at jsr166e.ForkJoinTask.doExec(ForkJoinTask.java:260)
    at jsr166e.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:858)
    at jsr166e.ForkJoinPool.scan(ForkJoinPool.java:1687)
    at jsr166e.ForkJoinPool.runWorker(ForkJoinPool.java:1642)
    at jsr166e.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:109)
```

Not sure what causes it though... it happens a few moments after opening my project, presumably during (or shortly after) re-indexing the project.

**Comment by ceis** [comment]
I get the same exception every time I try to organize the imports, but also on other occasions. For example, the plugin can no longer determine whether an import is used or not.

Happens since update to IntelliJ Ultimate 13.1.

**Comment by cefolger** [comment]
Thanks guys, I'll look into this and release a patch if necessary.

**Comment by cefolger** [comment]
There was a change in the API that required a minor fix (see 46affe8744aa81f883689ae312318031245246b7). I also tested this fix on 12.x and it worked fine. (Side note: I removed some tests that are no longer compiling against 12.x for this release, but will fix them on the main branch) 

I'll post this to the repository over the weekend. However, feel free to grab the 0.6.1 jar directly and use the 'install from disk' option in IntelliJ. 
https://github.com/cefolger/needsmoredojo/tree/0.6.1_dev/dist
